dockerfile_build/Dockerfile.ubuntu_build

```diff
@@ -0,0 +1,10 @@
+FROM bazel/reproducible:ubuntu
+RUN apt-get update -y && \
+    apt-get install --no-install-recommends -y -q \
+      build-essential \
+      python2.7 && \
+    apt-get clean && \
+    rm /var/lib/apt/lists/*_*
+
+# TODO(aaron-prindle) see the proper way to setup python to build node
+RUN ln -s /usr/bin/python2.7 /usr/bin/python
```

Comment:
You can drop this now that you have the alias.

Reply:
done